README.md

```diff
@@ -213,20 +216,22 @@ collections:
   - name: redhat.trusted_profile_analyzer
 ```
 
-or with the classic ansible command
+Or by using the following Ansible commands:
 
 ```shell
 ANSIBLE_ROLES_PATH="roles/" ansible-playbook -i inventory.ini play.yml -vv
```

Comment:
Is this a command a user can copy and paste as-is? Does the ANSIBLE_ROLES_PATH variable have to be exported first? Should it be like this?

export ANSIBLE_ROLES_PATH="roles/" ; ansible-playbook -i inventory.ini play.yml -vv

Reply:
ok